components/templates/HomeTemplate.tsx

```diff
@@ -95,8 +96,8 @@ const HomeTemplate = (): React.ReactElement => {
             view="20"
           />
         </>)}
-
```

Comment:
Stylelist?

Reply:
I don't quite understand what you mean. Explain further.

Reply:
implement the style list into the home template below the combo list.